asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js

```diff
@@ -56,100 +56,93 @@ class AssetTransfer extends Contract {
             },
         ];
 
-        for (let i = 0; i < assets.length; i++) {
-            assets[i].docType = 'asset';
-            await ctx.stub.putState(assets[i].ID, Buffer.from(JSON.stringify(assets[i])));
-            console.info('Added <--> ', assets[i]);
+        for (const asset of assets) {
+            asset.docType = 'asset';
+            await ctx.stub.putState(asset.ID, Buffer.from(JSON.stringify(asset)));
+            console.info(`Asset ${asset.ID} initialized`);
         }
     }
 
-    // createAsset issues a new asset to the world state with given details.
-    async createAsset(ctx, id, color, size, owner, appraisedValue) {
+    // CreateAsset issues a new asset to the world state with given details.
+    async CreateAsset(ctx, id, color, size, owner, appraisedValue) {
         const asset = {
             ID: id,
             Color: color,
             Size: size,
             Owner: owner,
             AppraisedValue: appraisedValue,
```

Comment:
can we make asset JSON compatible with that of go-chaincode ( where we have camelcase json keys like `appraisedValue`), so the application-* can work with any of these chaincodes.

specially because network.sh installs the chaincode-* using the same short name `basic` for all the langs.

Reply:
Will update in a follow on PR

Reply:
Though this doesn't break anything, the application works just fine with all of the chaincodes